deploy/docker/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM centos:7
+LABEL maintainers="Kubernetes Authors"
+LABEL description="RBD CSI Plugin"
+
+ENV CEPH_VERSION "luminous"
+RUN rpm -Uvh https://download.ceph.com/rpm-$CEPH_VERSION/el7/noarch/ceph-release-1-1.el7.noarch.rpm && \
```

Comment:
EPEL could cause CI trouble later, try this
```
   yum  install -y centos-release-ceph && \
     yum install -y ceph-common && 
```

Reply:
done, switched to centos-release-ceph